src/main/java/io/nats/client/api/DeliverPolicy.java

```diff
@@ -24,9 +24,10 @@
     Last("last"),
     New("new"),
     ByStartSequence("by_start_sequence"),
-    ByStartTime("by_start_time");
+    ByStartTime("by_start_time"),
+    LastPerSubject("last_per_subject");
```

Comment:
Not enough familiar with the Java lib: is that enough for the marshal/unmarshal of this new policy?

Reply:
yeah, when going to json, the to string returns policy string (the quoted value). When reading from json, we compare to the policy string.